### PR TITLE
Add NinjaMC

### PR DESCRIPTION
**Pull request by Tamino112**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/NinjaMC/manifest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "NinjaMC",
+  "nice_name": "NinjaMC.net",
+  "direct_ip": "NinjaMC.net",
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "Discord": "discord.gg/ninjamc"
```
Comment (DoJapHD) [inline minecraft_servers/NinjaMC/manifest.json:9]: JSON conventions, key has to be lowercase

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/NinjaMC/manifest.json (hunk 1 above)
Comment (Tamino112) [reply minecraft_servers/NinjaMC/manifest.json]: I changed it

Comment (Tamino112) [reply minecraft_servers/NinjaMC/manifest.json]: I changed it now

File: minecraft_servers/NinjaMC/manifest.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "NinjaMC",
+  "nice_name": "NinjaMC.net",
+  "direct_ip": "NinjaMC.net",
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "discord": "https://social.dc.ninjamc.net"
```
Comment (DoJapHD) [inline minecraft_servers/NinjaMC/manifest.json:9]: Discord link has to be https://discord.gg/[YOUR INVITE]
Custom links are for partner servers only

File: minecraft_servers/NinjaMC/manifest.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "NinjaMC",
+  "nice_name": "NinjaMC.net",
+  "direct_ip": "NinjaMC.net",
```
Comment (DoJapHD) [inline minecraft_servers/NinjaMC/manifest.json:4]: IP please in lowercase

File: minecraft_servers/NinjaMC/manifest.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "NinjaMC",
```
Comment (DoJapHD) [inline minecraft_servers/NinjaMC/manifest.json:2]: server_name please in lowercase (folder name, too!)

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/NinjaMC/manifest.json (hunk 4 above)
Comment (Tamino112) [reply minecraft_servers/NinjaMC/manifest.json]: i changed it

**Pull request closed without merging**